package/pythonapi-coco/.cm/meta.json

```diff
@@ -0,0 +1,32 @@
+{
+  "customize": {
+    "extra_dir": "", 
+    "install_env": {
+      "PACKAGE_BRANCH": "master", 
+      "PACKAGE_NAME": "pythonapi-coco", 
+      "PACKAGE_URL": "https://github.com/cocodataset/cocoapi.git", 
+      "TOOL_NAME": "pythonapi-coco"
```

Comment:
I'd recomment to use `tool-cocoapi` or simply `tool-coco`. We don't use `pythonapi-*` anywhere else.